### PR TITLE
Add message for displaying for eny email each time

### PR DESCRIPTION
**Pull request by bscheshirwork**
Propose changes for displaying message undependent of check email result. 

If this wiil be accepted we can use specific error type and set logging as optional. Also we can cath only specific error type, all another cathing by yii2 build-in hooks

| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes
| Breaks BC?    | no (but we need translate message on 10 languages. Loook like https://www.nvidia.com/fi-fi/account/forgot-password where fi-fi is country code) 
| Tests pass?   | yes
| Fixed issues  | #238

### CODE REVIEW

**Comment by tonydspaniard** [comment]
@bscheshirwork please always add the credits of your work at `CHANGELOG.md` please 🍻

File: src/User/resources/i18n/ru/usuario.php [hunk 1]
```diff
@@ -81,6 +81,7 @@
     'Account settings' => 'Настройки аккаунта',
     'Already registered? Sign in!' => 'Уже зарегистрированы? Войдите!',
     'An email has been sent with instructions for resetting your password' => 'Вам отправлено письмо с инструкциями по смене пароля',
+    'An email with instructions to create a new password has been sent to {email} if it is associated with an {appName} account. Your existing password has not been changed.' => 'Сообщение с инструкциями по созданию нового пароля было отправлено на адрес {email}, если он привязан к учетной записи {appName}. Ваш текущий пароль изменен не был.',
```
Comment (tonydspaniard) [inline src/User/resources/i18n/ru/usuario.php]: I believe we will need to update all translation files.

Comment (bscheshirwork) [reply src/User/resources/i18n/ru/usuario.php]: I not speak at 10 languages :cry:

Comment (TonisOrmisson) [reply src/User/resources/i18n/ru/usuario.php]: not translate...
run the ./yii message path/to/config.php command to update the new untranslated string to all of the language files.

Comment (bscheshirwork) [reply src/User/resources/i18n/ru/usuario.php]: *That* update... OK I will try it now

**Comment by tonydspaniard** [comment]
@bscheshirwork do you mind fixing the conflicts? 

BTW: Thanks for such amount of work and dedication to this library. Appreciated.

**Comment by bscheshirwork** [comment]
memo
```
docker-compose -f yii2-usuario-docker-composition.yml run --rm --entrypoint bash php
bash-4.4# ./vendor/yiisoft/yii2/yii message ./src/User/resources/i18n/config.php
```

regenerate message again...
So... `'sort' => false,` is correct settings? Oldest messages is sorted

**Pull request merged**